.github/workflows/yaml.yml

```diff
@@ -0,0 +1,16 @@
+name: Validate YAML
+
+on:  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [main]
```

Comment:
Why not validate on branches pushed as well?

Reply:
We can; as configured this only validates branches which are pull requests.